workers/ohsome_quality_analyst/geodatabase/client.py

```diff
@@ -214,33 +169,37 @@ async def get_bpolys_from_db(
             'features', json_agg(
                 json_build_object(
                     'type',       'Feature',
-                    'id',         ogc_fid,
+                    'id',         {fid_field},
                     'geometry',   public.ST_AsGeoJSON(geom)::json,
                     'properties', json_build_object(
-                        -- list of fields
-                        'fid', ogc_fid
                     )
                 )
             )
         )
-        FROM {0}
-        WHERE ogc_fid = $1
+        FROM {dataset}
+        WHERE {fid_field} = $1
     """
-    ).format(dataset)
+    ).format(fid_field=fid_field, dataset=dataset)
+
     async with get_connection() as conn:
-        result = await conn.fetchrow(query, feature_id)
+        try:
+            result = await conn.fetchrow(query, feature_id)
+        except (UndefinedColumnError, DataError):
+            # TODO: Do we need a custom error here?
+            # DataError occurs if feature id is a wrong type. E.g. str not int
+            raise
```

Comment:
Your are right. I removed the tr except clause. Those error should not be encountered anymore since dataset and fid_field are checked beforehand if they are valid